app/services/persona_detail_extraction.py

```
@@ -0,0 +1,70 @@
+            messages=[
+                {"role": "system", "content": "You are a highly skilled AI assistant capable of extracting structured information from text."},
+                {"role": "user", "content": llm_prompt}
+            ],
+            temperature=0.1, # Low temperature for precise extraction
+            max_tokens=500 # Limit token usage
+        )
+
+        content = response.choices[0].message.content
+        if content:
+            extracted_data = json.loads(content)
+            
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)